bootstrap.sh

```diff
@@ -154,14 +154,14 @@ if [ -z "$MYSQL_FLAVOR" ]; then
 fi
 case "$MYSQL_FLAVOR" in
   "MySQL56")
```

Comment:
Thanks for the clarification.

The test.go could be renamed to -image instead of -flavor for example?

After all, I'm fine with the current state and don't think we need to take any actions now.